specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "description": "Gets an Log Search rule",
+        "tags": [
+          "scheduledQueryRules"
+        ],
+        "operationId": "getScheduledQueryRule",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get a Log Search rule",
+            "schema": {
+              "$ref": "#/definitions/LogSearchRuleResource"
+            },
+            "examples": {
+              "application/json": {
+                "id": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/scheduledQueryRules/logalertfoo",
+                "name": "logalertfoo",
+                "type": "Microsoft.Insights/scheduledQueryRules",
+                "location": "West US",
+                "tags": {
+                  "hidden-link:/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/scheduledQueryRules/logalertfoo": "Resource"
+                },
+                "properties": {
+                  "description": "log alert description",
+                  "Enabled": "true",
+                  "skuType": "L1",
+                  "lastUpdatedTime": "2017-06-23T21:23:52.0221265Z",
+                  "provisioningState": "Succeeded",
+                  "source": {
+                    "query": "Type=HearBeat",
+                    "queryType": "ResultCount",
+                    "datasourceId": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/workspaces/servicedeskeus"
+                  },
+                  "schedule": {
+                    "frequencyInMinutes": 15,
+                    "timeWindowInMinutes": 15
+                  },
+                  "action": {
+                    "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.Microsoft.AppInsights.Nexus.DataContracts.Resources.ScheduledQueryRules.AlertingAction",
+                    "severity": "1",
+                    "status": "Inactive",
+                    "trigger": {
+                      "thresholdOperator": "GreaterThan",
+                      "threshold": 3
+                    },
+                    "aznsAction": {
+                      "actionGroup": [],
+                      "emailSubject": "Email Header",
+                      "customWebhookPayload": "{}"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
```

Comment:
Error states should be modeled as `"default"` responses and have a model to deserialize the error into.

A lot of these things should have been caught in the API review board -- I'll pause the review here, and wait until after the review board has seen these APIs.